tools/kagami/src/main.rs

```diff
@@ -692,7 +692,7 @@ mod tokens {
     #[derive(StructOpt, Debug, Clone, Copy)]
     pub struct Args;
 
-    pub fn permission_token_definitions() -> Result<[PermissionTokenDefinition; 17]> {
+    pub fn permission_token_definitions() -> Result<[PermissionTokenDefinition; 20]> {
```

Comment:
I wonder if Rust has means of inferring the size. `const` generic maybe?

Reply:
I'm not sure that const generic will work here because the size it's not something we pass inside function.

Reply:
Looks like there is a nightly feature: [rust-lang/rust/#85077](https://github.com/rust-lang/rust/issues/85077)

Reply:
IMO we can just switch to `Vec` since this place isn't performance critical at all.

Reply:
And I can't make it const or static, because of `token::Definition::with_params()`

Reply:
Fixed using `Vec`